src/main/java/net/coderbot/iris/pipeline/ShadowRenderer.java

```diff
@@ -148,48 +117,33 @@ public ShadowRenderer(WorldRenderingPipeline pipeline, ProgramSource shadow, Pac
 		//       relative to the player camera, not the shadow camera, so we can't rely on chunks being properly
 		//       sorted in the shadow pass.
 		//
+		// Note: Blending is automatically applied on 1.17 by our ExtendedShader.
+		//
 		// - https://github.com/IrisShaders/Iris/issues/483
 		// - https://github.com/IrisShaders/Iris/issues/987
-		BlendModeOverride blendModeOverride;
 
 		if (shadow != null) {
-			//this.shadowProgram = createProgram(shadow, directives, flipped);
-
-			blendModeOverride = shadow.getDirectives().getBlendModeOverride();
-
-			if (blendModeOverride == null) {
-				blendModeOverride = BlendModeOverride.OFF;
-			}
```

Comment:
Don't forget to replicate this behavior

Reply:
No need, this is handled automatically as part of ExtendedShader. Or at least it seems so?

Reply:
No, the behavior where the blend mode override defaults to OFF instead of no override for shadow programs is not handled by ExtendedShader.

Reply:
Done.